### PR TITLE
Edits to Logging 4.0 topics per team review 2

### PR DESCRIPTION
**Pull request by mburke5678**
Second round of edits from review. 
https://docs.google.com/document/d/1-E1z-Z_CvIC7u4ggQFfaH-9aF_p7lIGsdOLD7Zynj9g/edit?ts=5c759927#heading=h.c7x2dmk5jv1x

### CODE REVIEW

File: logging/efk-logging-configuring.adoc [hunk 1]
```diff
@@ -17,30 +17,132 @@ and includes all the components of the logging stack to collect, store and visua
...
+  creationTimestamp: '2019-03-20T18:07:02Z'
+  generation: 1
+  name: instance
+  namespace: openshift-logging
+  resourceVersion: '80645'
+  selfLink: >-
+    /apis/logging.openshift.io/v1alpha1/namespaces/openshift-logging/clusterloggings/instance
+  uid: f6195f55-4b3a-11e9-8da0-0ed069924724
 spec:
-  managementState: "Managed"
```
Comment (anpingli) [inline logging/efk-logging-configuring.adoc]: Shall we remove the openshift appended items, the line 27-30, and 65-145?

File: modules/efk-logging-elasticsearch-add-remove.adoc [hunk 2]
```diff
@@ -46,13 +46,13 @@ For example, to scale up your cluster:
...
 nodes:
-  - nodeCount: 2
+  - nodeCount: 4
     nodeSpec:
       resources: {}
     storage: {}
 ----
 
-. To scale down, create a Cluster Logging Custom Resource (CR) to reduce the number of nodes of a specific type:
+. To scale down, create a Cluster Logging Custom Resource (CR) to reduce the number of nodes:
```
Comment (anpingli) [inline modules/efk-logging-elasticsearch-add-remove.adoc]: @mburke5678  scale down is not supported.

File: modules/efk-logging-elasticsearch-add-remove.adoc [hunk 3]
```diff
@@ -35,7 +35,7 @@ metadata:
   name: "clusterlogging"
 ...
 nodes:
-  - nodeCount: 2 <1>
+  - nodeCount: 3 <1>
```
Comment (anpingli) [inline modules/efk-logging-elasticsearch-add-remove.adoc]: @mburke5678  There are some limitation about the Elasticsearch node number when using Cluster Logging custom Resource (CR). 
1) The CLO only create ES nodes using "roles": ["client","data","master"]. 
2) The nodeCount should be <=3.     as Elasticsearch Operator don't allow the ES master nodes number > 3

To customize Elasticsearch nodes type. Turn to CLO Unmanage status, and modify the elasticsearch CR directly
For example:  Add two dataonly ES node to cluster. You can use the following Elasticsearch CR
apiVersion: logging.openshift.io/v1alpha1
kind: Elasticsearch
metadata:
  name: elasticsearch
  namespace: openshift-logging
spec:
  managementState: Managed
  nodeSpec:
    image: quay.io/openshift/origin-logging-elasticsearch5:latest
  nodes:
  - nodeCount: 2
    roles:
    - client
    - data
    - master
    storage:
      size: 10Gi
      storageClassName: gp2
  - nodeCount: 2
    roles:
    - data
    storage:
      size: 10Gi
      storageClassName: gp2
  redundancyPolicy: ZeroRedundancy

File: modules/efk-logging-elasticsearch-add-remove.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/efk-logging-elasticsearch-add-remove.adoc]: @anpingli If the user scales up the number of nodes, can he scale back down to the original number?

File: modules/efk-logging-elasticsearch-add-remove.adoc (hunk 3 above)
Comment (mburke5678) [reply modules/efk-logging-elasticsearch-add-remove.adoc]: @anpingli Will the default memory settings allow for adding two data-only nodes, as you sow here? (I would test this myself. But, I am not running the default memory and the pods were not getting created due to insufficient memory.)

**Comment by mburke5678** [comment]
@anpingli I am going to merge this PR to prevent more merge errors. I am concerned with losing changes from this and other PRs due to these merge errors. 
If you have further comments, I can create a follow-up PR.

**Comment by anpingli** [comment]
@mburke5678  merge it. I like to review the document in a index mode rather than github.

**Pull request merged**